File: 1491. Average Salary Excluding the Minimum and Maximum Salary/Solution.java
class Solution {
    public double average(int[] salary) {
        int min = salary[0];
        int max = salary[0];
        int n = salary.length;

        for (int i = 0; i < n; i++) {
            if(min>salary[i]) min = salary[i];
            if(max<salary[i]) max = salary[i];
        }

        int sum = 0;
        for (int i = 0; i < n; i++) {
            if(salary[i] > min && salary[i]<max){
                sum += salary[i];
            }
        }

        return sum / n;

    }
}

File: 1491. Average Salary Excluding the Minimum and Maximum Salary/Solution.py
import math

class Solution:
    def average(self, salary: List[int]) -> float:
        _min = math.inf
        _max = -math.inf
        _sum = 0.0

        for s in salary:
            _min = min(s,_min)
            _max = max(s,_max)
            _sum += s

        return (_sum - _min - _max) / (len(salary) -2)